Sources/swift-doc/Subcommands/Generate.swift

```diff
@@ -74,9 +74,12 @@ extension SwiftDoc {
           case is Class, is Enumeration, is Structure, is Protocol:
             pages[route(for: symbol)] = TypePage(module: module, symbol: symbol, baseURL: baseURL, includingChildren: symbolFilter)
           case let `typealias` as Typealias:
-            pages[route(for: `typealias`.name)] = TypealiasPage(module: module, symbol: symbol, baseURL: baseURL)
+            pages[route(for: `typealias`.name)] = TypealiasPage(module: module, symbol: symbol, baseURL: baseURL, includingOtherSymbols: symbolFilter)
           case is Operator:
-            pages[route(for: symbol)] = OperatorPage(module: module, symbol: symbol, baseURL: baseURL)
+            let operatorPage = OperatorPage(module: module, symbol: symbol, baseURL: baseURL, includingImplementations: symbolFilter)
+            if !operatorPage.implementations.isEmpty {
```

Comment:
Is the thinking here that operators without implementations should be considered private/reserved and therefore documented? Or that we don't handle this case well?